userdocs/profiles.dev/docs/tutorial-basics/setup.md

```diff
@@ -42,16 +42,34 @@ version regularly updated.
 
 ### Profiles CRDs and Flux CRDs
 
-While profiles can be installed manually, it is recommended to install them in a GitOps fashion.
-Profiles uses Flux to do this.
+Profiles relies on Flux to deploy artifacts to your cluster, this means that at a minimum
+you much have the following Flux CRDs installed:
 
-To install both the profiles CRD and the required Flux CRDs, run:
+- [`helmreleases.helm.toolkit.fluxcd.io`](https://github.com/fluxcd/helm-controller/blob/main/config/crd/bases/helm.toolkit.fluxcd.io_helmreleases.yaml)
+- [`gitrepositories.source.toolkit.fluxcd.io`](https://github.com/fluxcd/source-controller/blob/main/config/crd/bases/source.toolkit.fluxcd.io_gitrepositories.yaml)
+- [`helmrepositories.source.toolkit.fluxcd.io`](https://github.com/fluxcd/source-controller/blob/main/config/crd/bases/source.toolkit.fluxcd.io_helmrepositories.yaml)
+- [`kustomizations.kustomize.toolkit.fluxcd.io`](https://github.com/fluxcd/kustomize-controller/blob/main/config/crd/bases/kustomize.toolkit.fluxcd.io_kustomizations.yaml)
+
+You can either install these manually by downloading and applying the above linked CRD
```

Comment:
those are just the CRDs, not the controller which we always need. I don't know why we'd document any way to install other than `flux install` :thinking: ?

Reply:
yep, was not sure, will delete